### ISSUE TITLE
Argument Passing when running main.py

### ISSUE THREAD

**Issue by aq737** [KaiyangZhou/deep-person-reid#137]
main.py: error: the following arguments are required: --root, -s/--sources

**Comment by KaiyangZhou** [comment]
you have to specify the root path, like `--root path/to/your/data-folder`

**Comment by WangGodder** [comment]
'Cython evaluation (very fast so highly recommended) is '
usage: main.py [-h] [--app {image,video}] [--loss {softmax,triplet}] --root
               ROOT -s SOURCES [SOURCES ...] [-t TARGETS [TARGETS ...]]
               [-j WORKERS] [--split-id SPLIT_ID] [--height HEIGHT]
               [--width WIDTH] [--train-sampler TRAIN_SAMPLER] [--combineall]
               [--random-erase] [--color-jitter] [--color-aug]
               [--seq-len SEQ_LEN] [--sample-method SAMPLE_METHOD]
               [--pooling-method {avg,max}] [--cuhk03-labeled]
               [--cuhk03-classic-split] [--use-metric-cuhk03]
               [--market1501-500k] [--optim OPTIM] [--lr LR]
               [--weight-decay WEIGHT_DECAY] [--momentum MOMENTUM]
               [--sgd-dampening SGD_DAMPENING] [--sgd-nesterov]
               [--rmsprop-alpha RMSPROP_ALPHA] [--adam-beta1 ADAM_BETA1]
               [--adam-beta2 ADAM_BETA2] [--max-epoch MAX_EPOCH]
               [--start-epoch START_EPOCH] [--batch-size BATCH_SIZE]
               [--fixbase-epoch FIXBASE_EPOCH]
               [--open-layers OPEN_LAYERS [OPEN_LAYERS ...]] [--staged-lr]
               [--new-layers NEW_LAYERS [NEW_LAYERS ...]]
               [--base-lr-mult BASE_LR_MULT] [--lr-scheduler LR_SCHEDULER]
               [--stepsize STEPSIZE [STEPSIZE ...]] [--gamma GAMMA]
               [--label-smooth] [--margin MARGIN]
               [--num-instances NUM_INSTANCES] [--weight-t WEIGHT_T]
               [--weight-x WEIGHT_X] [-a ARCH] [--no-pretrained]
               [--load-weights LOAD_WEIGHTS] [--evaluate]
               [--eval-freq EVAL_FREQ] [--start-eval START_EVAL]
               [--dist-metric DIST_METRIC] [--ranks RANKS [RANKS ...]]
               [--print-freq PRINT_FREQ] [--seed SEED] [--resume PATH]
               [--save-dir SAVE_DIR] [--use-cpu] [--gpu-devices GPU_DEVICES]
               [--use-avai-gpus] [--visrank] [--visrank-topk VISRANK_TOPK]
main.py: error: the following arguments are required: --root, -s/--sources

Process finished with exit code 2
 how to fix this problem, Thanks

**Comment by KaiyangZhou** [comment]
You need to give inputs to `--root` and `-s`, e.g. `python script.py --root path/to/reid-data -s market1501`

**Comment by victorcarv16** [comment]
> Você precisa fornecer informações para `--root`e `-s`, por exemplo,`python script.py --root path/to/reid-data -s market1501`


How do I do that?
